app/api/endpoints/webhooks/zalo_webhook.py

```
@@ -0,0 +1,37 @@
+    dependencies=[Depends(verify_zalo_webhook_signature)],
+    status_code=200,
+)
+async def handle_zalo_webhook(request: Request) -> dict[str, str]:
+    """Handle POST /api/webhooks/zalo after signature verification.
+
+    Body is validated with ``parse_zalo_webhook_payload`` against the
+    ``ZaloWebhookPayload`` discriminated union in ``app.schemas.webhooks.zalo_schema``.
+    """
+    try:
+        data = await request.json()
+    except Exception:
```

Ruff (0.15.11)

[warning] Do not catch blind exception: `Exception`

(BLE001)